notebook/tests/selenium/test_shutdown.py

```diff
@@ -0,0 +1,13 @@
+"""Tests shutdown of the Kernel."""
+
+def test_shutdown(notebook):
+    notebook.edit_cell(content="print(21)")
+    notebook.browser.find_element_by_xpath('//a[text()="Kernel"]').click()
+    notebook.browser.find_element_by_id('shutdown_kernel').click()
+    notebook.browser.find_element_by_class_name('btn.btn-default.btn-sm.btn-danger').click()
+
+    notebook.wait_explicitly(3)
```

Comment:
Is it possible to do this without a time.sleep? Instead, perhaps wait for a condition? Sleep with a timer leaves a race condition in slow environments.

Reply:
Sure, you're right. Instead of `time.sleep` I will create a method which uses one of the wait ones, so its time to wait can be flexible. I will send the fix, thank you for reviewed it 👍

Reply:
@minrk let me know if it needs more refactor, please : D